MessengerOnFileForLinux/UserService/test/SignOutTestCore.hpp

```diff
@@ -0,0 +1,25 @@
+#pragma once
```

Comment:
Po co cała ta klasa skoro nic z tym nie robisz? Nie lepiej w takim razie zrobić z tego po prostu zwykły test file?

Reply:
Nie chciałem robić w każdym teście inicjalizcji SignOut no i jeszcze wrzuciłem tu funkcje signInUser(). Uważam że przez to testy są dużo czytelniejsze i od razu widać co robią.

Reply:
ok, w takim razie jest drobna uwaga poniżej